vertical-pod-autoscaler/pkg/recommender/model/aggregate_container_state.go

```diff
@@ -109,6 +113,8 @@ type AggregateContainerState struct {
 	IsUnderVPA          bool
 	UpdateMode          *vpa_types.UpdateMode
 	ScalingMode         *vpa_types.ContainerScalingMode
+	OomBumpUpRatio      float64
```

Comment:
Can we have the vars and getters consistent with the other var, please?
```suggestion
	OOMBumpUpRatio      float64
```